casper/src/main/scala/coop/rchain/casper/dag/BlockDagKeyValueStorage.scala

```diff
@@ -177,6 +177,15 @@ final class BlockDagKeyValueStorage[F[_]: Concurrent: Log] private (
         _ <- finalizationEffect(indirectlyFinalized + directlyFinalizedHash)
         // persist finalization
         _ <- blockMetadataIndex.recordFinalized(directlyFinalizedHash, indirectlyFinalized)
+
+        // TODO: temp code to satisfy tests
```

Comment:
Just curious which tests it is trying to fix?

Reply:
This test specifically, but maybe I seen some other test failing also.
https://github.com/rchain/rchain/blob/0ad4974f751f1be0a7d706dc07ff7a635041716c/casper/src/test/scala/coop/rchain/casper/dag/BlockDagKeyValueStorageTest.scala#L230-L301